ci/dockerfiles/ansible-e2e-hybrid.Dockerfile

```
@@ -23,20 +23,9 @@ ENV OPERATOR=/usr/local/bin/ansible-operator \
 # Ensure fresh metadata rather than cached metadata in the base by running
 # yum clean all && rm -rf /var/yum/cache/* first
 RUN yum clean all && rm -rf /var/cache/yum/*
```

Need a trailing backslash at the end of this line:

```suggestion
RUN yum clean all && rm -rf /var/cache/yum/* \
```

Ditto in other Dockerfiles

---

:-( my bad.